jib-core/src/main/java/com/google/cloud/tools/jib/image/json/HistoryObjectTemplate.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 Google LLC. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.image.json;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.google.cloud.tools.jib.json.JsonTemplate;
+import java.util.Objects;
+import javax.annotation.Nullable;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class HistoryObjectTemplate implements JsonTemplate {
+
+  /** The timestamp at which the image was created. */
+  private String created;
+
+  /** The name of the author specified when committing the image. */
+  @Nullable private String author;
+
+  /** The command used while building the image. */
+  @Nullable private String created_by;
+
+  /** Whether or not the layer is empty */
+  @Nullable private Boolean empty_layer;
```

Comment:
`boolean`? I believe we can default to `false`.

Reply:
I wanted to make this field optional, so I made it a `@Nullable Boolean` instead of `boolean` (if it's `boolean` then the field will always show up in the generated config).

Reply:
Ah right, that makes sense :+1: